clouddrift/__init__.py

```diff
@@ -9,5 +9,4 @@
 import clouddrift.ragged
 import clouddrift.signal
 import clouddrift.sphere
-import clouddrift.plotting
```

Comment:
Is it not required so that we can do

```
import clouddrift as cd
cd.plotting.plot_ragged(...)
```

?

Reply:
I added a `try: except:` ...